virtio-devices/src/balloon.rs

```diff
@@ -542,17 +542,23 @@ impl VirtioDevice for Balloon {
         &mut self,
```

Comment:
Nit: Typo from the commit message of "virtio-devices: Pass a list of tuples for virtqueues": `related to *teh* queue`

Reply:
Thanks!